src/Styleguide/Containers/Artist/AuctionResults/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import { graphql, QueryRenderer } from "react-relay"
+import { ContextProps, ContextConsumer } from "Components/Artsy"
+import { Subscribe } from "unstated"
+
+import { FilterState } from "./state"
+
+import Container from "./AuctionResults"
+
+interface Props extends ContextProps {
+  artistID: string
+}
+
+class ArtistFilter extends React.Component<Props> {
```

Comment:
This filename and named export can be a little more descriptive and should match 1:1 unless additional exports from this file are needed (in which the filename can be more general):

```jsx
export const AuctionResultsFilter = ContextConsumer(props => {
  const { artistID, relayEnvironment } = props

  return (
    <Subscribe to={[FilterState]}>
      ...
    </Subscribe>
  )
})
```

Reply:
It also seems like `/Containers` should exist within our namespace as opposed to putting everything top-level -- https://github.com/artsy/reaction/tree/master/src/Styleguide/Pages/Artist.

Later I think we can refactor this organization further, since the container will essentially be root of the tab / page (but that kind of cleanup can come in a few rounds)